package.json

```diff
@@ -15,6 +15,7 @@
     "babel-preset-env": "^1.6.1",
     "babel-register": "^6.26.0",
     "browser-sync": "^2.23.6",
+    "chart.js": "^2.7.1",
```

Comment:
No need for this line since you're using the CDN version of chart.js. Remove with `yarn uninstall chart.js`